src/grb_advroutines.jl

```diff
@@ -0,0 +1,53 @@
+mutable struct GRBsvec
+    len::Cint
+    ind::Ptr{Cint}
+    val::Ptr{Cdouble}
+end
+
+function get_tableaurow_grb(model::Model, cidx::Integer)
+	len = num_vars(model)+num_constrs(model)
+	idx = Array{Cint}(len)
+	val = Array{Cdouble}(len)
```

Comment:
Looks like the indentation is off?

Reply:
What is the default here? Spaces or tabs?

Reply:
4 spaces